templates/.VERSION

```diff
@@ -1 +1 @@
-0.9.99.0915
\ No newline at end of file
+v0.9.99-212-g2c92e45
```

Comment:
Currently, if I clone and dev feature, Gitea will never run because i'm not necessary check out at commit g2c92e45.